Book.js

```diff
@@ -0,0 +1,44 @@
+const saveToLocalStorage = (key, data) => localStorage.setItem(key, JSON.stringify(data));
+const getFromLocalStorage = (key) => JSON.parse(localStorage.getItem(key));
+
+class Book {
+  constructor(title, author, id = undefined) {
+    this.title = title;
+    this.author = author;
+    if (id !== undefined) {
+      this.id = id;
+    }
+  }
+
+  getCode() {
+    return `<div>${this.title}</div>
+              <div>${this.author}</div>
+              <button class='remove' data-id='${this.id}'>Remove</button>
+              <hr>`;
+  }
+
+  static add(book) {
+    const bookItems = this.getAllBooks();
+    let id = 1;
+    if (bookItems.length > 0) {
+      id = bookItems[bookItems.length - 1].id + 1;
+    }
+    book.id = id;
+    bookItems.push(book);
```

Comment:
- [ ] OPTIONAL
It will be a good practice to favor the use of array methods such as `concat` over `push`, which doesn't mutate the array.